spec/features/canvas_display_spec.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+describe 'Canvas Displaly', type: :feature do
+  let(:exhibit) { FactoryGirl.create(:exhibit, slug: 'default-exhibit') }
```

Comment:
@jkeck can you remove `FactoryGirl`? This `create` should still work, and it'll mean less work in #852.